### PR TITLE
Add index.js

### PR DESCRIPTION
**Pull request by mezlet**

### CODE REVIEW

File: nibss/node/index.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+
+var username = 'Your organization code';
+//Converting organization code to base 64
+var organisation_code = Buffer.from(username).toString("base64");
+var sandbox_key = "Your sandbox key";
+
+//setting request headers
+var reset_options = {
+  hostname: "innovation-sandbox-backend.herokuapp.com",
```
Comment (akinjide) [inline nibss/node/index.js]: Make hostname an empty string

File: nibss/node/index.js [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    "Sandbox-Key": sandbox_key,
+    OrganisationCode: organisation_code
+  },
+  method: "POST"
+};
+
+
+//Reset Sandbox Credentials
+function Reset() {
+var data = "";
```
Comment (akinjide) [inline nibss/node/index.js]: indent inward

File: nibss/node/index.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+  };
+  const req = http.request(reset_options, callback).end();
+}
+
+Reset();
+
+//Data gotten from Reset() headers
+const aes_key = 'Your Aes key';
+const password = "Your password";
+const ivkey = 'Your ivkey';
```
Comment (akinjide) [inline nibss/node/index.js]: Automatically set these values from resp.headers on line 32

File: nibss/node/index.js [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+const crypto = require('crypto');
+
+
+var username = 'Your organization code';
+//Converting organization code to base 64
+var organisation_code = Buffer.from(username).toString("base64");
+var sandbox_key = "Your sandbox key";
+
+//setting request headers
+var reset_options = {
```
Comment (akinjide) [inline nibss/node/index.js]: move reset_options declaration into `function Reset()` scope

File: nibss/node/index.js [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  .digest("hex");
+
+ //Converting to base64 
+const authHeader = Buffer.from(`${username}:${password}`).toString("base64");
+const signatureMethodHeader = "SHA256";
+const bvn = '12345678901';
+const encrypted =  encrypt(`{"BVN": "${bvn}"}`);
+
+const options = {
+  hostname: "innovation-sandbox-backend.herokuapp.com",
```
Comment (akinjide) [inline nibss/node/index.js]: Make hostname an empty string

File: nibss/node/index.js [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+    },
+    method: "POST",
+};
+
+
+//Verify BVN
+function BVN() {
+  let data = "";
+  let decrypted;
+  const req = http.request(options, res => {
```
Comment (akinjide) [inline nibss/node/index.js]: consistently use callback like in `function Reset()`

**Review by akinjide — CHANGES_REQUESTED** [review]
Improve code indentation and address comments

File: nibss/node/index.js (hunk 1 above)
Comment (mezlet) [reply nibss/node/index.js]: done

File: nibss/node/index.js (hunk 2 above)
Comment (mezlet) [reply nibss/node/index.js]: done

File: nibss/node/index.js (hunk 3 above)
Comment (mezlet) [reply nibss/node/index.js]: done

File: nibss/node/index.js (hunk 4 above)
Comment (mezlet) [reply nibss/node/index.js]: done

File: nibss/node/index.js (hunk 5 above)
Comment (mezlet) [reply nibss/node/index.js]: done

File: nibss/node/index.js (hunk 6 above)
Comment (mezlet) [reply nibss/node/index.js]: done

**Comment by akinjide** [comment]
@mezlet Looks good. Add instruction to set hostname as comment

**Pull request merged**